source/geometry_model/ellipsoidal_chunk.cc

```diff
@@ -0,0 +1,795 @@
+    EllipsoidalChunk<dim>::declare_parameters(ParameterHandler &prm)
+    {
+      prm.enter_subsection("Geometry model");
+      {
+        prm.enter_subsection("Ellipsoidal chunk");
+        {
+          prm.declare_entry("NE corner",
+                            "",
+                            Patterns::Anything(),
+                            "Longitude:latitude in degrees of the North-East corner point of model region."
+                            "The North-East direction is positive. If one of the three corners is not provided"
+                            "the missing corner value will be calculated so all faces are parallel.");
+          prm.declare_entry("NW corner",
+                            "",
+                            Patterns::Anything(),
+                            "Longitude:latitude in degrees of the North-West corner point of model region. "
+                            "The North-East direction is positive. If one of the three corners is not provided"
+                            "the missing corner value will be calculated so all faces are parallel.");
+          prm.declare_entry("SW corner",
+                            "",
+                            Patterns::Anything(),
+                            "Longitude:latitude in degrees of the South-West corner point of model region. "
+                            "The North-East direction is positive. If one of the three corners is not provided"
+                            "the missing corner value will be calculated so all faces are parallel.");
+          prm.declare_entry("SE corner",
+                            "",
+                            Patterns::Anything(),
+                            "Longitude:latitude in degrees of the South-East corner point of model region. "
+                            "The North-East direction is positive. If one of the three corners is not provided"
+                            "the missing corner value will be calculated so all faces are parallel.");
+          prm.declare_entry("Depth",
+                            "500000.0",
+                            Patterns::Double(0),
+                            "Bottom depth of model region.");
+          prm.declare_entry("Semi-major axis",
+                            "6378137.0",
+                            Patterns::Double(0),
+                            "The semi-major axis (a) of an ellipsoid. This is the radius for a sphere (eccentricity=0). Default WGS84 semi-major axis.");
+          prm.declare_entry("Eccentricity",
+                            "8.1819190842622e-2",
+                            Patterns::Double(0),
+                            "Ellipticity of the ellipsoid. Zero is a perfect sphere, default (8.1819190842622e-2) is WGS84.");
+          prm.declare_entry("East-West subdivision",
+                            "1",
+                            Patterns::Integer(0),
+                            "The number of subdivisions in the East-West direction.");

```

Comment:
say something like "The number of subdivisions of the coarse (initial) mesh in East-West direction" to make clear that it's about subdivisions of the *mesh*. Same below.